tools/rvn/CommandLineApp.cs

```diff
@@ -51,6 +61,92 @@ public static int Run(string[] args)
+                        {
+                            case ImportCertificateMode when string.IsNullOrEmpty(pathToCertVal):
+                                ExitWithError("--path option must be set",cmd);
+                                break;
+                            case ImportCertificateMode:
+                                return await SetupPackageConfiguratorLetsEncrypt(setupParamVal, packageOutParamVal, cmd, ImportCertificateMode, pathToCertVal, certPassVal, token);
+                            case LetsEncryptMode when string.IsNullOrEmpty(setupParamVal) == false:
+                                return await SetupPackageConfiguratorLetsEncrypt(setupParamVal, packageOutParamVal, cmd, LetsEncryptMode, pathToCertVal, certPassVal, token);
+                            default:
+                                ExitWithError("--mode and --setup-params options must be set",cmd);
+                                break;
+                        }
+                       
+                        return 0;
+
+                    });
+            });
+        }
+
+        private static async Task<int> SetupPackageConfiguratorLetsEncrypt(string setupParamsPath, string packageOutParam, CommandLineApplication cmd, string mode, string certPath, string certPass, CancellationToken token)
+        {
+            var zipFile = Array.Empty<byte>();
+            
+            using (StreamReader file = File.OpenText(setupParamsPath ?? string.Empty))
+            {
+                var progress = new SetupProgressAndResult(null)
+                {
+                    Processed = 0,
+                    Total = 2
+                };
+                
+                JsonSerializer serializer = new();
+                var setupInfo = (SetupInfo)serializer.Deserialize(file, typeof(SetupInfo));
+                
+                if (setupInfo != null)
+                {
+                    if (packageOutParam == null)
+                        packageOutParam = setupInfo.Domain + "-package.zip";
+                    else
+                        packageOutParam += "-package.zip";
+
+                    if (mode is ImportCertificateMode)
+                    {
+                        var certBytes = await File.ReadAllBytesAsync(certPath, token);
+                        var certBase64 = Convert.ToBase64String(certBytes);
+                        
+                        setupInfo.Certificate =  certBase64;
+                        setupInfo.Password = certPass;
+                        zipFile = await LetsEncryptRvnUtils.ImportCertificateSetup(setupInfo, setupParamsPath, progress, token);
+                    }
+                    else
+                    {
+                        zipFile = await LetsEncryptRvnUtils.SetupLetsEncrypt(setupInfo, setupParamsPath, progress, string.Empty ,token);
+                    }
+                }
+                var path = Path.Combine(AppContext.BaseDirectory, packageOutParam);
```

Comment:
Instead of changing using app's base dir, we should use what's passed in `packageOutParam` alone. User would in general expect that if he run this in his home dir with
`/opt/ravendb/RavenDB/Server/rvn -o /tmp/setup_package.zip`

The resulting package would be in `/tmp/setup_package.zip`.
The app should not care where we're at the moment. Also if a relative path is specified e.g. `./package.zip` then just save it there.

Reply:
done